ExeterTeam25/group-software-project
# Delete comments

What I did: 
- Made it so users can delete their own comments 

To test: 
- Make sure you have at least 2 comments, each by a different person 
- Log in as one of these people 
- Click view comments 
- You should be able to delete only your own comments, try to make sure it works
